app/helpers/save_this_page_helper.rb

```diff
@@ -27,6 +27,7 @@ def link_href(options)
 
   def additional_text(options)
     signed_in = options[:signed_in] || false
-    I18n.t(SEE_SAVED_PAGES_LOGGED_IN[signed_in], link: "/account/saved-pages")
+    additional_class = options[:additional_class] || ""
+    I18n.t(SEE_SAVED_PAGES_LOGGED_IN[signed_in], link: "#{Plek.new.website_root}/account/saved-pages", additional_class: additional_class)
```

Comment:
Is this meaningfully different to

    I18n.t(SEE_SAVED_PAGES_LOGGED_IN[signed_in], link: "#{Plek.new.website_root}/account/saved-pages", additional_class: options[:additional_class])

?

Reply:
I thought the line above was necessary but seems like it's not!